app/models/user_system_notification.rb

```diff
@@ -16,6 +18,16 @@ def self.mark_as_read(notification_id)
     end
   end
 
+  def self.unseen_count
+    unseen.count
+  end
+
+  def self.modal(notification_id)
```

Comment:
Didn't you move this to scope?

Reply:
I think maybe remove at all, cause we have unseen scope. And if we need find unseen count, we can simply use unseen.count.

Reply:
@aignatov-bio Yes, that's a great idea.